x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -358,6 +359,43 @@ export function trainedModelsRoutes({ router, routeGuard }: RouteInitialization)
     })
   );
 
+  /**
+   * @apiGroup TrainedModels
+   *
+   * @api {post} /api/ml/trained_models/infer/:modelId Evaluates a trained model
```

Comment:
This looks like a copy-and-paste from another route.

Can you add it to `apidoc.json` too please.